tests/test_main.py

```diff
@@ -465,16 +428,12 @@ def test_movie_session_service_get_movies_sessions(database_data):
 @pytest.mark.django_db
 def test_movie_session_service_get_movie_session_by_date(database_data):
     sessions_1 = get_movies_sessions("2019-8-19")
```

Comment:
The `get_movies_sessions` function expects a `datetime` object for the `session_date` parameter, but a string is provided here. Consider converting the string to a `datetime` object before passing it to the function.